### PR TITLE
[bitnami/airflow] Take worker.resources in values.yaml into consideration, when using KubernetesExecutor

### LINKED ISSUE #6279: [bitnami/airflow] resources ignored for worker in KubernetesExecutor
**Which chart**:
Airflow

**Which version**:
2.0.2-debian-10-r7

**Describe the bug**
I don't know if this is intentional, but when using KubernetesExecutor ```worker.resources``` in ```values.yaml``` is ignored. 
This is a problem for instance in openshift, when administrators have set up default limits for containers, because then you would regularly run out of memory.

**To Reproduce**
Just start a worker and verify the YAML file that is generated when starting a worker with KubernetesExecutor - you will see that ```worker.resources```  is empty, even if you have defined it in your values.yaml

**Expected behavior**
I would expect resources settings in ```values.yaml``` to be taken into consideration, or at least get a warning that when using KubernetesExecutor it will be ignored.
The fix is easy, if this is not intentional. Just include in ```template/config/configmap.yml``` the resources

```
 containers:
        - name: airflow-worker
          image: {{ include "airflow.workerImage" . }}
          imagePullPolicy: {{ .Values.worker.image.pullPolicy }}
        {{- if .Values.containerSecurityContext.enabled }}
          securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
        {{- end }}
          {{- if .Values.worker.resources }}
          resources: {{- toYaml .Values.worker.resources | nindent 12 }}
          {{- end }}
          env:
```

### PR DESCRIPTION
**Pull request by diegoami**
**Description of the change**

Ensure that `worker.resources` specified in `values.yaml` are taken into consideration also when spinning a new worker

**Benefits**

Limits and requests in `worker.resources` have an effect 

**Possible drawbacks**

None

**Applicable issues**

  - fixes #6279

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/airflow/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 annotations:
   category: WorkFlow
 apiVersion: v2
-appVersion: 2.0.2
+appVersion: 2.0.3
```
Comment (juan131) [inline bitnami/airflow/Chart.yaml:4]: You shouldn't bump the `appVersion` but `version` instaed.

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for this fix!! Please take a look to my comment

File: bitnami/airflow/Chart.yaml (hunk 1 above)
Comment (diegoami) [reply bitnami/airflow/Chart.yaml]: Ok, done

**Review by juan131 — APPROVED** [review]
Thanks!

**Pull request merged**